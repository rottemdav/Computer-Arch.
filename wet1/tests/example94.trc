4 4 25 2 local_history global_tables using_share_lsb
0x53a9fe8 N 0x83d8fc
0x7614e14 T 0x5e1cd454
0xdd8 N 0x814
0xde27cc T 0xc44
0x7e9b017c N 0xfd3410c0
0x4ac0 T 0x430
0x8bb8 T 0x04a75c
0x6a8 N 0xc5575c
0x63a0347c N 0x7b82b848
0x7ed45c T 0x2cdf4
0x0a7a8a70 N 0x63a04
0x3d55d8 N 0x587bc
0x3d55d8 T 0x587bc
0xde27cc T 0xc44
0xa26490 T 0x96664
0x318 T 0x028
0x0a7a8a70 T 0x63a04
0x7fc80 T 0x1b4d8
0x0a7a8a70 N 0x63a04
0x7614e14 T 0x5e1cd454
0xde27cc N 0xc44
0xd7cc N 0xf611bc
0xdd8 T 0x814
0x7fc80 N 0x1b4d8
0x0e3fc60 T 0x30fa0
0x56c T 0x915330
0x318 T 0x028
0x56c T 0x915330
0x8bb8 T 0x04a75c
0x0a7a8a70 T 0x63a04
0x7614e14 T 0x5e1cd454
0x0a7a8a70 T 0x63a04
0xce2510 T 0xb0f8
0x3ad914 N 0xf9e30
0x7614e14 N 0x5e1cd454
0x4ac0 T 0x430
0x4ac0 N 0x430
0x7614e14 N 0x5e1cd454
0x3ad914 T 0xf9e30
0x0e3fc60 N 0x30fa0
0xa26490 T 0x96664
0xce2510 T 0xb0f8
0xf6c N 0x08c
0x094e3a9c N 0xb3e74
0x7e9b017c T 0xfd3410c0
0x3b800ab0 N 0xac86614
0x0e3fc60 N 0x30fa0
0x7e9b017c T 0xfd3410c0
0x318 T 0x028
0x7614e14 T 0x5e1cd454
0x63a0347c T 0x7b82b848
0x63a0347c T 0x7b82b848
0x8eda0 T 0xef4
0xaf1d924 T 0xa2615c
0x7ed45c N 0x2cdf4
0x3d55d8 N 0x587bc
0x7614e14 T 0x5e1cd454
0x7614e14 N 0x5e1cd454