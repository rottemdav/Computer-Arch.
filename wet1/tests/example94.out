0x53a9fe8 N 0x53a9fec
0x7614e14 N 0x7614e18
0xdd8 N 0xddc
0xde27cc N 0xde27d0
0x7e9b017c N 0x7e9b0180
0x4ac0 N 0x4ac4
0x8bb8 N 0x8bbc
0x6a8 N 0x6ac
0x63a0347c N 0x63a03480
0x7ed45c N 0x7ed460
0xa7a8a70 N 0xa7a8a74
0x3d55d8 N 0x3d55dc
0x3d55d8 N 0x3d55dc
0xde27cc N 0xde27d0
0xa26490 N 0xa26494
0x318 N 0x31c
0xa7a8a70 N 0xa7a8a74
0x7fc80 N 0x7fc84
0xa7a8a70 N 0xa7a8a74
0x7614e14 T 0x5e1cd454
0xde27cc T 0xc44
0xd7cc N 0xd7d0
0xdd8 N 0xddc
0x7fc80 N 0x7fc84
0xe3fc60 N 0xe3fc64
0x56c N 0x570
0x318 N 0x31c
0x56c N 0x570
0x8bb8 N 0x8bbc
0xa7a8a70 N 0xa7a8a74
0x7614e14 T 0x5e1cd454
0xa7a8a70 T 0x63a04
0xce2510 N 0xce2514
0x3ad914 N 0x3ad918
0x7614e14 N 0x7614e18
0x4ac0 N 0x4ac4
0x4ac0 T 0x430
0x7614e14 N 0x7614e18
0x3ad914 N 0x3ad918
0xe3fc60 N 0xe3fc64
0xa26490 N 0xa26494
0xce2510 N 0xce2514
0xf6c N 0xf70
0x94e3a9c N 0x94e3aa0
0x7e9b017c N 0x7e9b0180
0x3b800ab0 N 0x3b800ab4
0xe3fc60 N 0xe3fc64
0x7e9b017c T 0xfd3410c0
0x318 N 0x31c
0x7614e14 N 0x7614e18
0x63a0347c N 0x63a03480
0x63a0347c T 0x7b82b848
0x8eda0 N 0x8eda4
0xaf1d924 N 0xaf1d928
0x7ed45c N 0x7ed460
0x3d55d8 N 0x3d55dc
0x7614e14 N 0x7614e18
0x7614e14 T 0x5e1cd454
flush_num: 33, br_num: 58, size: 272b
